### PR TITLE
Apply Symbiotic Entity to rollAction Melee Attacks

### PR DESCRIPTION
**Pull request by Aeristoka**

### CODE REVIEW

**Comment by kakaroto** [comment]
That whole block is inside ` if ( isMeleeAttack || action_name.includes("Lightning Launcher")) {`
So shouldn't this check for symbiotic Entity be `if (isMeleeAttack && symbiotic....)` ? Or should it also apply on Lightning Launcher action?

**Comment by Aeristoka** [comment]
> That whole block is inside ` if ( isMeleeAttack || action_name.includes("Lightning Launcher")) {`
> So shouldn't this check for symbiotic Entity be `if (isMeleeAttack && symbiotic....)` ? Or should it also apply on Lightning Launcher action?

Good catch, fixed now.

File: src/dndbeyond/content-scripts/character.js [hunk 1]
```diff
@@ -898,6 +898,12 @@ function rollAction(paneClass, force_to_hit_only = false, force_damages_only = f
             settings_to_change["charger-feat"] = false;
         }
 
+        // Druid: Circle of Spores - Symbiotic Entity
+        if (isMeleeAttac && character.hasClassFeature("Symbiotic Entity") && character.getSetting("druid-symbiotic-entity", false)) {
```
Comment (kakaroto) [inline src/dndbeyond/content-scripts/character.js:902]: You clearly didn't test this 🙄  (type in variable name 😝)

Comment (Aeristoka) [reply src/dndbeyond/content-scripts/character.js]: Not sure what you're talking about 😇

**Pull request merged**